src/lib/wavtools/lib/wav_stream_player.js

```diff
@@ -119,23 +121,17 @@ export class WavStreamPlayer {
     } else {
       throw new Error(`argument must be Int16Array or ArrayBuffer`);
     }
-  
     // Example: Process smaller chunks of the buffer
     const processChunk = async (startIdx, endIdx) => {
       const chunkBuffer = buffer.slice(startIdx, endIdx);
-  
-      // Convert Int16Array to AudioBuffer
-      const audioBuffer = await int16ArrayToAudioBuffer(chunkBuffer, this.sampleRate);
-  
-      // Resample to 24,000 Hz
-      const resampledBuffer = await resampleAudioBuffer(audioBuffer, 24000);
-  
-      // Convert the resampled AudioBuffer back to Int16Array
+      const audioBuffer = await int16ArrayToAudioBuffer(chunkBuffer, apiSampleRate); // Use apiSampleRate directly
+      const resampledBuffer = await resampleAudioBuffer(audioBuffer, apiSampleRate); 
```

Comment:
Shouldn't this be the audio context rate?
```ts
      const resampledBuffer = await resampleAudioBuffer(audioBuffer, this.samplerate);
```
On Firefox, *not* doing this results in a squeaky voice - but also stuttering. Then again, I'm a terrible tester.

Reply:
all good! I haven't had the time to fully finish this pull request